CMakeLists.txt

```diff
@@ -90,7 +90,7 @@ else()
     include(FetchContent)
     FetchContent_Declare(glfont
       GIT_REPOSITORY https://github.com/ami-iit/GLFont
-      GIT_TAG 68914f7093acb485c01fe661e7b4b2cb91921bef)
+      GIT_TAG f31bac175c233fa4028d0883cda91f6db7995b38)
```

Comment:
Thanks for dealing with this. I think I made a mistake in merging https://github.com/ami-iit/GLFont/pull/2. I did want to keep the same hash, so I did "Squash and Merge", but I guess I should have merged normally.